laser_filters_plugins.xml

```diff
@@ -18,6 +18,12 @@
 	This is a filter which filters sensor_msgs::LaserScan messages based on range
       </description>
     </class>
+    <class name="laser_filters/LaserScanAngularIntervalFilter" type="laser_filters::LaserScanAngularIntervalFilter" 
+      base_class_type="filters::FilterBase<sensor_msgs::LaserScan>">
+      <description>
+  This is a filter which filters sensor_msgs::LaserScan messages based on angular intervals.
```

Comment:
Could you add a couple more sentences to the description? Maybe something like `For example, if the intervals parameter is ...., then the filter will only allow laser beams whose angle is between .... and ....`